scripts/publish-pct-shiny

```diff
@@ -10,7 +10,7 @@ VAR_DIR="/home/git/var-shiny/"
 WAIT_SECONDS=0
 BRANCH=$1
 SHINY_APP_DIR="pct-shiny"
-DATA_GIT="pct-outputs-regional-R"
+DATA_GIT="pct-outputs-regional-Rsmall"
```

Comment:
This will expect the `pct-outputs-regional-Rsmall` git repo to already exist on the server so will fail.  I don't think we will change the data repo much so I don't intent to automate this but if we do want to merge it we will have to run the commands in the `Server.md` file manually first.